packages/svelte/tests/runtime-runes/samples/lifecycle-render-order-for-children-3/_config.js

```diff
@@ -8,13 +8,13 @@ export default test({
 
 	async test({ assert, component, logs }) {
 		assert.deepEqual(logs, [
-			'parent: render 0',
 			'1: $effect.pre 0',
 			'1: render 0',
 			'2: $effect.pre 0',
 			'2: render 0',
 			'3: $effect.pre 0',
 			'3: render 0',
+			'parent: render 0',
```

Comment:
Why is the parent template now happening before all the children templates?

Reply:
It's the opposite. Previously you could have multiple template effects in the root fragment, and the `{logRender()}` call happened before rendering the child. Now, there's just one template effect, at the end (after rendering child blocks and components etc). I don't consider this a breaking change since we're already grouping some effects at the end without giving you any control over it, and since template effects should be pure it's effectively not observable